apps/expo-workflow-testing/.eas/workflows/updates-e2e-custom-init.yml

```
@@ -27,18 +27,11 @@ jobs:
     runs_on: macos-large
     image: latest
     steps:
-      - name: Install applesimutils
-        id: ios_simulator
-        env:
-          HOMEBREW_NO_AUTO_UPDATE: "1"
-        run: |
-          /opt/homebrew/bin/brew tap wix/brew
-          /opt/homebrew/bin/brew install applesimutils
-          xcrun simctl list
+      - uses: eas/install_maestro
       - uses: eas/checkout
       - uses: eas/use_npm_token
       - uses: eas/install_node_modules
-      - name: Set up Updates E2E custom init project
+      - name: Set up Updates E2E old arch project
```

```suggestion
      - name: Set up Updates E2E custom init project
```

is this custom init?